### ISSUE TITLE
Method of producing Bezier Curves

### ISSUE THREAD

**Issue by ph3rro** [luncheon/simplify-svg-path#3]
Is the logic used in producing bezier curves similar to what is presented in https://arxiv.org/pdf/1902.10525.pdf?

The site is https://arxiv.org/abs/1902.10525

I noticed, while using react-native-draw, that it overfits lines a bit, and makes it a bit too precise for my liking, as I need the fewest number of curves possible when inputting to a neural network model. For example, when I try to draw a line, it gets represented as multiple connected bezier curves, whereas I would like it to be represented as one curve, despite the slight loss of precision. Are there variables I can adjust to make it simpler like this?

Thanks

**Comment by ph3rro** [comment]
Ah, so it's the tolerance variable. I have a new question now.
